hr_timesheet_limit_sale_date/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Simone Orsi <simone.orsi@camptocamp.com>
```

Comment:
Is it a new module ? If so did they all contribute ?

Reply:
they contribute in previous dev

Reply:
ok